.github/workflows/build.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
         run: |
           echo "IMAGE_TAG=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
       - name: Checkout
-        uses: actions/checkout@v4.1.0
+        uses: actions/checkout@v4.1.1
```

Comment:
Rather than going to `v4.1.1`, shouldn't this have been generalized to `v4`?

Ditto line 52.